### PR TITLE
Update Readme to include all ways to get involved

### PR DESCRIPTION
**Pull request by Malwine**
This PR updates our Readme to include all ways to get involved into ClojureBridge and wasdiscussed with @yokolet and @annapawlicka

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,55 @@
-There are a lot of different ways to organize a workshop, but if you've got the [Minimum Viable Workshop](/Minimum-Viable-Workshop.md), you'll have a great time. There are way more people in the world who are excited to teach and learn Clojure than you know.
+# Get involved
 
-The [Workshop Planning Timeline](Workshop-Planning-Timeline.md) is a good place to start; it's divided into sections before, during, and after the workshop. There are *a lot* of individual tasks, so we recommend organizing in a team.
+ClojureBridge aims to increase diversity within the Clojure community by offering free, beginner-friendly Clojure programming workshops for women.
 
-We have past organizers who can help you figure out what to do. Email info@clojurebridge.org and we'll introduce you to an organizing mentor!
+ClojureBridge was started in 2014 and is part of [Bridge Foundry](http://bridgefoundry.org/). All around the world there have been workshops organized by people like you! Many people volunteered in different ways. Here are some insights for you to get involved:
```
Comment (Malwine) [inline README.md]: @yokolet When was ClojureBridge founded? 2014 or earlier?

File: README.md [hunk 2]
```diff
@@ -1,13 +1,55 @@
...
 
-ClojureBridge follows the BridgeFoundry [Code of
-Conduct](http://bridgefoundry.org/code-of-conduct/).
+There are different ways you can contribute. Maybe you like to...
+
+###...organize!
+
+You want to organize a workshop in your area? Great! Let’s get started! For a [minimal viable workshop](https://github.com/ClojureBridge/organizing/blob/master/Minimum-Viable-Workshop.md) you need to find: 
+
+- **A venue:** A safe and quiet space for learners to do their first steps in Clojure. An office leisure space or a co-working space might work.
```
Comment (davidchambers) [inline README.md]: I suggest changing _do their first steps_ to _take their first steps_.

File: README.md [hunk 3]
```diff
@@ -1,13 +1,55 @@
...
+- **A date:** ClojureBridge is a 2 day workshop. Which means you need to find a date for the install party (e.g. Friday evening from 5PM - 8PM) and a date for the workshop day (e.g. Saturday 9 AM - 6 PM).
```
Comment (davidchambers) [inline README.md]: I suggest _two-day_ in place of _2 day_.

Let's make the spacing around "PM" consistent. :stuck_out_tongue_winking_eye:

File: README.md [hunk 4]
```diff
@@ -1,13 +1,55 @@
...
+
+You can find lots of information in our [organizing repository](https://github.com/ClojureBridge/organizing) on GitHub and read about the [Workshop Planning Timeline](Workshop-Planning-Timeline.md). [Resources on Diversity](Resources-on-Diversity.md) provides a number of great resources on how to create a diverse, inclusive event. Feel free to read and contribute to these documents.
+
+ClojureBridge follows the BridgeFoundry [Code of Conduct](http://bridgefoundry.org/code-of-conduct/).
+
+Last, but not least, you might want to find some sponsors. They can help you provide awesome food, drinks or fancy swag for your learners so that this workshop will stay a memorable event.
+
+###...coach!
+
+As a coach you and your co-coach will teach a small group of attendees Clojure and fundamental programming principles at the workshop.This way you can help to start new carriers in tech. Have a look at our curriculum. Coaching is an awesome experience. It helps you understand Clojure better and to practice your explaining skills which is also useful for work.
```
Comment (davidchambers) [inline README.md]: Nit: We're missing a space between the first and second sentences.

File: README.md [hunk 5]
```diff
@@ -1,13 +1,55 @@
...
+
+###...be a teaching assistant!
+
+As a teaching assistant or a “floating coach” you also help attendees learn Clojure. You are not assigned to a fixed group but you can float around and help when questions arise. Being a teaching assistant might be your thing if you just started learning Clojure yourself. 
+
+###...sponsor or donate!
+
+What would ClojureBridge be without our sponsors? Many great companies and private people have sponsored ClojureBridge workshops. With their help it was possible to provide food, drinks, T-Shirts, notebooks and stickers for the attendees at different workshops.
```
Comment (davidchambers) [inline README.md]: I suggest _T-shirts_ rather than _T-Shirts_.

File: README.md [hunk 6]
```diff
@@ -1,13 +1,55 @@
...
+As a teaching assistant or a “floating coach” you also help attendees learn Clojure. You are not assigned to a fixed group but you can float around and help when questions arise. Being a teaching assistant might be your thing if you just started learning Clojure yourself. 
+
+###...sponsor or donate!
+
+What would ClojureBridge be without our sponsors? Many great companies and private people have sponsored ClojureBridge workshops. With their help it was possible to provide food, drinks, T-Shirts, notebooks and stickers for the attendees at different workshops. 
+You can help by sponsoring the workshop in your area or abroad but also a donation to ClojureBridge is very welcome to help us spread the word, run our servers etc.
+
+###...help improve the curriculum!
+
+Our curriculum can be found in our curriculum repository. You are very welcome to join our curriculum mailing list and share your ideas.
```
Comment (davidchambers) [inline README.md]: Shall we make "curriculum repository" a link?

**Comment by davidchambers** [comment]
This is a terrific addition, @Malwine!

File: README.md (hunk 6 above)
Comment (Malwine) [inline README.md]: I actually had that in my draft and then forgot. Will put it in again. Thanks for the reminder. I will also add a few more links.

File: README.md (hunk 5 above)
Comment (Malwine) [inline README.md]: 🇺🇸

File: README.md (hunk 4 above)
Comment (Malwine) [inline README.md]: 👾

File: README.md (hunk 3 above)
Comment (Malwine) [inline README.md]: 2️⃣

File: README.md (hunk 2 above)
Comment (Malwine) [inline README.md]: 🏃

File: README.md (hunk 1 above)
Comment (Malwine) [inline README.md]: @davidchambers Do you have an idea?

Comment (davidchambers) [inline README.md]: 2014 sounds right to me, but @bridgethillyer is the source of truth on this matter. :)

Comment (bridgethillyer) [inline README.md]: Yep, 2014. RailsBridge was founded in 2009.

Comment (Malwine) [inline README.md]: Cool, thank you @bridgethillyer !

**Comment by Malwine** [comment]
@davidchambers Initially I wanted to update the /get-involved page for clojurebridge.org 
So this is the first step. I would love if we could somehow render this markdown file at http://clojurebridge.org/get-involved
Maybe there is a way.

**Comment by davidchambers** [comment]
> I would love if we could somehow render this markdown file at http://clojurebridge.org/get-involved

I'm not familiar with Rails, so I don't know what's involved. Presumably there's a way to provide HTML content to be included in a document. If this is the case, I'll happily write a script to fetch the Markdown document from GitHub and run it through pandoc or similar to generate the HTML content.

**Comment by yokolet** [comment]
I believe this pull request has been discussed and reviewed well. Since markdown is hard to look at on the local env, I'm going to merge this pull request. If someone finds the problem including styling stuff, let fix those after merging.

**Comment by yokolet** [comment]
Looks good on github website.

**Pull request merged**